mathml.css

```diff
@@ -19,8 +19,15 @@ math[display="block"] {
 /* fraction */
 mfrac {
     display: inline-block !important;
-    vertical-align: -50%;
+    vertical-align: 30%;
     border-collapse: collapse;
+    line-height: 110%;
+    font-size: 90%;
+}
+@media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+    mfrac {
+        vertical-align: -50%;
+    }
```

Comment:
Well, these are valid points, but before my fix styles work different in Chrome and IE11 (I haven't checked MS Edge, though), so I made styles more consistent.